k8s/aditum/chart/aditum/.helmignore

```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
```

Comment:
a `.DS_Store` file is included in this PR. Can you remove it and maybe update `.gitignore` to include this file type as well?

Reply:
Done!